### ISSUE TITLE
'and' instead of 'an'

### ISSUE THREAD

**Issue by melnibon** [realm/realm-java#5512]
Hi,

It seems to me there is a mistake in error message `Replacing and element is not supported.`

This message appear in my release at io.realm.internal.Collection$ListIterator.set(Collection.java:257)
```
public void set(T object) {
    throw new UnsupportedOperationException("Replacing and element is not supported.");
}
```

Best regards,
Pierre

**Comment by cmelchior** [comment]
Hi @melnibon Thank you. We should definetely fix that 👍

**Comment by Zhuinden** [comment]
Definitely? 😄

**Comment by cmelchior** [comment]
Fixed by #5513
